fastai/data/transforms.py

```diff
@@ -178,8 +178,9 @@ def __init__(self, pat, match=False):
         self.matcher = self.pat.match if match else self.pat.search
 
     def __call__(self, o):
-        res = self.matcher(str(o))
-        assert res,f'Failed to find "{self.pat}" in "{o}"'
+        posix_o = Path(o).as_posix()
```

Comment:
`Path` is really slow, and this function needs to be fast. So I'd suggest something like

```
import os,posixpath
o = str(o).replace(os.sep, posixpath.sep)
```